Formula/poppler.rb

```diff
@@ -58,6 +59,9 @@ class Poppler < Formula
   def install
     ENV.cxx11
 
+    # removes /usr/include from CFLAGS (not clear why)
+    ENV["PKG_CONFIG_SYSTEM_INCLUDE_PATH"] = "/usr/include" if MacOS.version < :mojave
```

Comment:
@Rylan12 is this still the way to do this?

Reply:
Yes, inside the `install` method the `if` statements are the way to go.

`brew style` has also been updated, now, so barring any weird bugs I think if the style checks pass the formula is in good shape (with respect to `on_system` vs `if` statements at least).